app/controllers/Authentication.scala

```diff
@@ -408,8 +408,8 @@ class Authentication @Inject()(
 
   private def creatingOrganizationsIsAllowed(requestingUser: Option[User]) = {
     val noOrganizationPresent = InitialDataService.assertNoOrganizationsPresent
-    val configurationFlagSet = Play.configuration.getBoolean("application.allowOrganzationCreation").getOrElse(false) ?~> "allowOrganzationCreation.notEnabled"
-    val userIsSuperUser = requestingUser.exists(_.isSuperUser).toFox
```

Comment:
Is it correct that `.toFox` and `bool2Fox` have the same behavior? The only difference is that Boolean does not have a method called `.toFox` thats why it is implicitly converted to a Fox and then returns itself, while `bool2Fox` converts it explicitly?

Reply:
`.toFox` is actually a method of Fox. This way, if called on any type _that has an implicit conversion to Fox_, that conversion will be triggered. This will now no longer work for booleans (as is intended).